### PR TITLE
Upgrade to HeroIcons v0.4.2

### PR DESCRIPTION
**Pull request by iksaku**
> Reference tree: [v0.4.2﻿](https://github.com/tailwindlabs/heroicons/tree/v0.4.2)

Also, added compatibility with `^0.4.0` release structure, which moved `outline/` and `solid/` from vcs root to `optimized/`.

### CODE REVIEW

File: bin/compile.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@
 set -e
 
 DIRECTORY=$(cd `dirname $0` && pwd)
-DIST=$DIRECTORY/../dist
+DIST=$DIRECTORY/../dist/optimized
```
Comment (driesvints) [inline bin/compile.sh:6]: Can you revert this change?

Comment (iksaku) [reply bin/compile.sh:6]: On heroicons’ repo, the new “root” for svg elements is in “optimized folder”, I thought it was ok to reference it as if a nested git repo in local.

Will revert

**Comment by driesvints** [comment]
Thanks! I'll try to release this soon.

**Comment by driesvints** [comment]
Released as 0.3.1: https://github.com/blade-ui-kit/blade-heroicons/releases/tag/0.3.1
